pkg/authorization/api/register.go

```diff
@@ -59,6 +77,52 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 
 		&RoleBindingRestriction{},
 		&RoleBindingRestrictionList{},
+	}
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		append(types,
+			&unversioned.Status{}, // TODO: revisit in 1.6 when Status is actually registered as unversioned
+			&kapi.ListOptions{},
+			&kapi.DeleteOptions{},
+			&kapi.ExportOptions{},
+		)...,
 	)
+	versioned.AddToGroupVersion(scheme, SchemeGroupVersion)
+	return nil
+}
+
+func addLegacyKnownTypes(scheme *runtime.Scheme) error {
+	types := []runtime.Object{
```

Comment:
We should not add anything into legacy types after this PR merges. I also don't want to share types because Kube is not doing that either and with this each package is effectively a standalone API group.